spec/acceptance/lastEvaluatedKeyWithFilterAndPagingTest.spec.ts

```diff
@@ -116,6 +116,22 @@ describe('lastEvaluatedKeyWithFilterAndPagingTest', function ()
         assert.equal(page3.items.length, 0);
         assert.isUndefined(page3.firstEvaluatedKey);
         assert.isUndefined(page3.lastEvaluatedKey);
+
+        let page1Again = await nodenamo.list().from(User).filter(filter).limit(2,2).order(false).resume(page2.firstEvaluatedKey).execute<User>();
+        assert.deepEqual(page1Again.items, [
+            user3,
+            user1
+        ]);
+        assert.isDefined(page1Again.firstEvaluatedKey)
+        assert.isUndefined(page1Again.lastEvaluatedKey)
+
+        let page2Again = await nodenamo.list().from(User).filter(filter).limit(2,2).resume(page1Again.firstEvaluatedKey).execute<User>();
+
+        assert.equal(page2Again.items.length, 2);
+        assert.deepEqual(page2Again.items[0], user4);
+        assert.deepEqual(page2Again.items[1], user5);
+        assert.isDefined(page2Again.firstEvaluatedKey);
+        assert.isDefined(page2Again.lastEvaluatedKey);
```

Comment:
Can we assert that page2Again.firstEvaluatedKey and page2Again.lastEvaluatedKey are equal to page2.firstEvaluatedKey and page2.lastEvaluatedKey respectively?